sos/report/plugins/aap_containerized.py

```diff
@@ -107,6 +107,19 @@ def setup(self):
                     subdir="aap_containers_log"
             )
 
+        if "automation-controller-task" in aap_containers:
+            container = "automation-controller-task"
+            podman_commands = [
+                (f"su - {username} -c 'podman exec -it {container} bash -c"
+                 " \"awx-manage check_license --data\"'",
+                 "awx-manage_check_license_--data"),
+                (f"su - {username} -c 'podman exec -it {container} bash -c"
+                 " \"awx-manage list_instances\"'",
+                 "awx-manage_list_instances_--data"),
```

Comment:
The filename does not match the command executed, which can be confusing for users. Worth removing `_--data` from the filename?